src/SkeletalTrapezoidation.h

```diff
@@ -51,6 +52,11 @@ class SkeletalTrapezoidation
     using edge_t = HalfEdge<SkeletalTrapezoidationJoint, SkeletalTrapezoidationEdge>;
     using node_t = HalfEdgeNode<SkeletalTrapezoidationJoint, SkeletalTrapezoidationEdge>;
     using Beading = BeadingStrategy::Beading;
+    using TransitionMiddle = SkeletalTrapezoidationEdge::TransitionMiddle;
+    using TransitionEnd = SkeletalTrapezoidationEdge::TransitionEnd;
+
+    template<typename T>
+    using temp_data_t = std::vector<std::shared_ptr<T>>;
```

Comment:
I'm not a big fan of this name. Sure, in all cases where you use this vector of pointers, you use it as a temporary data storage, but that's not what this structure is (It's just how you happen to use it, right?). 

That being said, I'm also not sure how we should name this. Perhaps vector_t?

Reply:
Ah I see, if we're going for a _type_ type, we can probably make it `ptr_vector_t` then.